### ISSUE TITLE
Booting into emergency mode after I upgraded the system

### ISSUE THREAD

**Issue by FrostedPenguin** [deadhead420/arch-linux-anywhere#441]
Hello.   
I successfully installed the system with arch-linux-anywhere and have used it without any problems.
Yesterday I updated entire system with pacman -Syu. After rebooting, the system automatically go into emergency mode. It said that system failed to load linux kernel and mount /boot/efi .
This is second time of exact same problem. At the first time I just fresh install the system but now I decided to fix it and make it work again.
I think I should make a booting disk and fix the problem with chroot. However I want to know what makes this problem so that I can fix the problem permanently  be prepared when this happened again. Do you have any idea?

**Comment by FrostedPenguin** [comment]
It turned out that /boot wasn't mounted properly when I update the kernel. 
So I made booting disk and mount the systen 
```
mount /dev/<system partition> /mnt 
mount /dev/<ESP partition> /mnt/boot/efi 
```
and chrooted into system with ` arch-chroot /mnt/bin/bash `

Finally `mkinitcpio -p linux` solve the problem.

But I'm still figuring out why /boot wasn't mounted before upgrade.

**Comment by pnedkov** [comment]
I am glad you solved this on your own. But I don't think it is an installer issue because you said the installation went successfully and broke **after** you updated everything. At least this is how I understand your words.

**Comment by FrostedPenguin** [comment]
Yes. I worried opening this issue because I think the installer looks fine and there is high chance that I ruined the system with my hands. However, I decided to report this issue cause it happened twice and at the first time when it happened, it was right after **the fresh install.** In addition to, I really didn't even touch things related to kernel, /boot partition  such as `/etc/fstab`. What I did was installing a few packages and fonts and nvidia driver through `pacman`. So at that time I thought there is something wrong in the installer. Those were the reason why I reported this issue. 

Sadly I'm still figuring out what caused the problem. Currently, I assume that the problem was happened by my mistake during installation process. I followed this [video](https://youtu.be/qAb_qL6IEMY).

Apparently this isn't that big issue. Sorry for bothering you :)
